src/csvcubed/cli/inspect/inspectdatasetmanager.py

```diff
@@ -99,13 +101,15 @@ def get_standard_shape_measure_col_name_from_dsd(
         ComponentPropertyAttributeURI.MeasureType.value,
     )
 
-    if len(filtered_components) == 0 or filtered_components[0].csv_col_title == "":
+    if len(filtered_components) == 0 or filtered_components[0].used_in_columns == "":
```

Comment:
This is wrong.

Reply:
that means on line 108 "return filtered_components[0].csv_col_title" has to be changed as well right?